website/content/en/preview/concepts/nodeclaims.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "NodeClaims"
+linkTitle: "NodeClaims"
+weight: 30
+description: >
+  Understand NodeClaims
+---
+
+Karpenter uses NodeClaims to perform node autoscaling based on the requests of unschedulable pods and the
```

Comment:
I would frame it as "Karpenter uses NodeClaims to manage the lifecycle of Kubernetes Nodes with the underlying cloud provider. Karpenter's algorithm creates and deletes NodeClaims in response to the demands of the Pods in the cluster.

Reply:
e.g., you could theorically create a standalone NodeClaim, separate from any autoscaler. Or build a "replica based" approach similar to MNG using NodeClaims.

Reply:
Done.